xep-0390.xml

```diff
@@ -51,6 +52,12 @@
     <jid>jonas@wielicki.name</jid>
   </author>
   <revision>
+    <version>0.1</version>
+    <date>2017-03-23</date>
+    <initials>jwi</initials>
+    <remark><p>Add security consideration for caps spamming.</p><p>Properly reference UTF-8 and Base64 RFCs.</p><p>Remove namespace prefixes from examples.</p><p>Properly reference &fullstop;.</p><p>Clarify the first error condition in the Hash Function Input algorithm.</remark>
```

Comment:
You forgot the last </p> here.  Also you should probably split them on multiple lines, and use <ul/> and <li/>.